### DISCUSSION TITLE
Request for GitHub Pro Student Access

### DISCUSSION THREAD

**Discussion by amirsajjadrahmani** [community/community#136709] [GitHub Education]
### Hi everyone,

Hi,

I am currently a PhD student at KU Leuven. My student card does not display a specific expiration date. Despite trying with all other available documents, GitHub continues to reject them. Could you please assist me in obtaining GitHub Pro as a student?
This is my official site from the university:
https://www.kuleuven.be/wieiswie/en/person/00170641

Kind regards,
Amir

**Comment by davevad93** [comment]
Hi @amirsajjadrahmani ,

Check the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) for more info about rejections.

You may also provide documentation other than your student ID, as indicated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student):

- ***Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.***

**Comment by ferid333** [comment]
Hi Amir,

It sounds frustrating that your documents keep getting rejected. Since your student card doesn’t show an expiration date, you might want to try submitting alternative documents like an official enrollment letter from your university or a transcript that includes your current academic year. These documents should help verify your student status.

If you haven’t already, include a link to your official university profile (like the one you provided) in your application, as it can help confirm your affiliation with KU Leuven.

If the rejections continue, you might want to reach out to GitHub Education support via their contact form, explaining your situation in detail.

Good luck, and I hope you can get your GitHub Pro as a student soon!
